content/guides/concepts/dnslink.md

```diff
@@ -7,70 +7,57 @@ menu:
 
 **NOTE: The info below is a quick guide to DNSLink. For a more complete guide, including tutorials, usage examples and FAQs, check out [dnslink.io](http://dnslink.io/).**
 
-## About DNSLink
+## What is DNSLink?
 
-DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map
-a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your
-DNS records, you can use them to always point to the latest version of an
-object in IPFS (remember that an IPFS object’s address changes if you modify
-the object). Because DNSLink uses DNS records, the names it produces are also
-usually easy to type and read.
+DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your DNS records, you can use them to always point to the latest version of an object in IPFS (remember that an IPFS object’s address changes if you modify the object). Because DNSLink uses DNS records, the names it produces are also usually easy to type and read.
```

Comment:
```suggestion
DNSLink uses [DNS TXT](https://en.wikipedia.org/wiki/TXT_record) records to map a domain name (like `ipfs.io`) to an IPFS address. Because you can edit your DNS records, you can use them to always point to the latest version of an object in IPFS (remember that an IPFS object’s address changes if you modify the object). Since DNSLink uses DNS records, you can assign names/paths/(sub)domains/whatever that are easy to type, read, and remember.
```